utils/simulations_v1_water_cycle.csv

```diff
@@ -0,0 +1,52 @@
+model_version, group, resolution, category, simulation_name, machine, checksum, experiment, ensemble_num, link_type, node,
```

Comment:
- I recall, for the v2 simulations, each simulation had a Confluence page that listed the values to use for `checksum`, but I don't know if `v1` had that too.
- It was a little easier to deduce the `experiment` & `ensemble_num` for `LR` simulations. What should the `HR` simulation `experiment`s be?
- What should the `link_type` (CMIP only, naitve, both?) be for these simulations?

Reply:
I don't think we had checksums. The machine is long gone, so we cannot reproduce these simulations anyway.